### DISCUSSION TITLE
Can't use custom API components after upgrading to `fumadocs-openapi@5.0.3`

### DISCUSSION THREAD

**Discussion by olabrahmi** [fuma-nama/fumadocs#698] [Q&A] [answered]
1. I've just upgraded to `fumadocs-openapi@5.0.3` and I'm unable to use some custom API components that I've built, such as my own `APIPlayground` component, the new `APIPage` component introduced by the new package version doesn't seem to allow for this.

2. As a side issue, my OpenAPI configuration is split into multiple files, this doesn't seem to be a case handled by the new `createOpenAPI` method from `fumadocs-openapi`. Passing a directory instead of a single OpenAPI yaml file results in the following error. Am I doing something wrong?

<img width="987" alt="Screenshot 2024-08-12 at 14 20 31" src="https://github.com/user-attachments/assets/05373471-a070-4563-8e9c-ee969162cf0e">

**Comment by fuma-nama** [comment, marked as answer]
1. You can still access the default components from `fumadocs-openapi/ui`, and override certain components with the `renderer` option of `createOpenAPI`.

2. Because we omitted the OpenAPI Schema path/url on generated MDX pages, `createOpenAPI` requires an OpenAPI Schema but doesn't support multiple schemas. I think opening a feature request for this would be more appropriate.
